pyocd/debug/svd/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""
+ mbed CMSIS-DAP debugger
+ Copyright (c) 2019 ARM Limited
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
```

Comment:
Would you mind changing the license header to use '#' comments instead of a docstring? The docstring shows up in Doxygen output. See `pyocd.trace.__init__` for an example you can copy. Thanks!